wrapper-cpp/modules/c_meeting_chat_interface.cpp

```
@@ -18,6 +18,59 @@ extern "C" ZOOMSDK::IChatMsgInfo *meeting_chat_build(
+    void *ptr_to_rust,
+    const char *message_id,
+    unsigned int sender_user_id,
+    const char *sender_display_name,
+    const char *content,
+    time_t timestamp
+);
+
+class C_MeetingChatCtrlEvent : public ZOOMSDK::IMeetingChatCtrlEvent {
+public:
+    C_MeetingChatCtrlEvent(void *ptr) {
+        ptr_to_rust = ptr;
```

Clang (14.0.6)

[warning] 'ptr_to_rust' should be initialized in a member initializer of the constructor

(cppcoreguidelines-prefer-member-initializer)